src/plot_utils.py

```diff
@@ -96,7 +102,18 @@ def heatmap(bdb, bql, **kwargs):
     """
     df = bqlu.cursor_to_df(bdb.execute(bql))
     df.fillna(0, inplace=True)
-    return zmatrix(df, **kwargs)
+    if selectors is None:
+        return zmatrix(df, **kwargs)
+    results = []
+    for n0selector in selectors:
+      n0selection = deps['name0'].map(n0selector)
+      for n1selector in selectors:
+        n1selection = deps['name1'].map(n1selector)
+        this_block = deps[n0selection & n1selection]
+        results.append(zmatrix(this_block, vmin=0, vmax=1))
+        plt.show()
```

Comment:
How will this play out in interactive but not IPython use?
Will it be possible to draw those plots headless and unattended and save them to files?